Traceback (most recent call last):
  File "main.py", line 32, in <module>
  File "main.py", line 22, in menuselection
RuntimeError: input(): lost sys.stdin
